docs/standard/serialization/system-text-json/how-to.md

```diff
@@ -85,15 +85,7 @@ The preceding examples use type inference for the type being serialized. An over
 :::code language="csharp" source="snippets/system-text-json-how-to/csharp/SerializeWithGenericParameter.cs" highlight="23":::
 :::code language="vb" source="snippets/system-text-json-how-to/vb/RoundtripToString.vb" id="SerializeWithGenericParameter":::
 
-With .NET 7, `System.Text.Json` supports serializing and deserializing <xref:System.DateOnly> and <xref:System.TimeOnly> types. Consider the following object:
-
-:::code source="snippets/system-text-json-how-to/csharp/RoundtripDateOnly.cs" id="SeriesDataPoint":::
-:::code source="snippets/system-text-json-how-to/vb/RoundtripDateOnly.vb" id="SeriesDataPoint":::
-
-The following example serializes the `SeriesDataPoint` object, displays the resulting JSON, and then deserializes it back into a new instance of the `SeriesDataPoint` type. Finally, the original and newly deserialized instances are compared for equality and the results are written to the console:
-
-:::code source="snippets/system-text-json-how-to/csharp/RoundtripDateOnly.cs" id="RoundtripDateOnly":::
-:::code source="snippets/system-text-json-how-to/vb/RoundtripDateOnly.vb" id="RoundtripDateOnly":::
+[!INCLUDE [dateonly-and-timeonly-serialization](../../datetime/includes/dateonly-and-timeonly-serialization.md)]
```

Comment:
With DateOnly and TimeOnly covered in the other articles, I don't think it needs to be covered here. It seems out of place here, where the focus isn't on specific types but on serialization and deserialization in general.

Reply:
Removed it, thanks.